### DISCUSSION TITLE
About "LunaScript"

### DISCUSSION THREAD

**Discussion by LoveBodhi** [Wohlstand/TheXTech#783] [General]
LunaScript is only for support of LunaDLL Episodes, instead of CREATION!
If removed, past episode/levels might be affected!

**Comment by Superbloxen** [comment]
I think LunaDLL shouldn't be _fully_ removed from the game but simply disabled by default in future versions, like how anticheat trap is disabled by default but can be enabled when building the game.

**Comment by Wohlstand** [comment]
I don't want to remove it too, basically, this is major thing I built and debugged by myself. Additionally, I am totally against REMOVES of functionality that will introduce limitation of playing of old content. Even this thing will be deprecated, it will still supported at least on mainstream platforms except low-end like 3DS in favour of performance/code-size.

**Reply by LoveBodhi** [reply]
... Should be only enabled on SMBX asset and packed games?
